backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/index/impl/ElasticsearchIndexManagerImpl.java

```diff
@@ -59,12 +60,14 @@
 		this.model = model;
 		this.refreshAfterWrite = refreshAfterWrite;
 		this.workPlanOrchestrator = indexingBackendContext.createWorkPlanOrchestrator( elasticsearchIndexName.encoded, refreshAfterWrite );
+		this.streamOrchestrator = streamOrchestrator;
 	}
 
 	@Override
 	public void close() {
 		// Index managers own the work plan context, but not the stream context (which is shared)
 		workPlanOrchestrator.close();
+		streamOrchestrator.close();
```

Comment:
If the first orchestrator throws an exception when closing, the second one will never get closed.
Please use a closer instead; see `org.hibernate.search.backend.elasticsearch.impl.ElasticsearchBackendImpl#close` for an example.